### PR TITLE
fix: correctly set default values for arrays :bubble_tea: 

### PR DESCRIPTION
**Pull request by jamacku**
now `[]` will override the default value.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -29,12 +29,20 @@ export class Config {
...
+    ) => configArray;
+
     const retrievedConfig = (
       await octokit.config.get({
         ...context.repo,
         path,
         defaults: configs =>
-          deepmerge.all([this.defaults, ...configs]) as Partial<Config>,
+          deepmerge.all([this.defaults, ...configs], {
+            arrayMerge: overwriteMerge,
```
Comment (phracek) [inline src/config.ts:44]: `arrayMerge` is not specified in config.ts, or is it part of TypeScript programming language?

**Review by phracek — CHANGES_REQUESTED** [review]
It seems like arrayMerge is not defined? I don't follow.

**Review by phracek — APPROVED** [review]
I see. it is used by deepmerge https://www.npmjs.com/package/deepmerge

File: src/config.ts (hunk 1 above)
Comment (phracek) [reply src/config.ts:44]: Now I have fount a definition.

Comment (jamacku) [reply src/config.ts:44]: It's a configuration option for the deepmerge library.

**Pull request merged**